src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (number) => {
```

Comment:
Consider adding a comment above the `adder` function to describe its behavior for future reference. It's always a good practice to document the purpose and usage of complex functions.